dist/decorated-stock-chart.js

```diff
@@ -344,6 +344,7 @@
                         },
                         changeTitle: function(title){
                             scope.states.chart.setTitle({text: title});
+                            scope.onDefaultAttributeChange({newAttr: scope.defaultSecurityAttribute});
```

Comment:
why? changeTitle just change the title of the chart - why should it be seen as a change of `defaultSecurityAttribute`